.emacs.d/lisp/code/editorconfig-c.el

```diff
@@ -0,0 +1,27 @@
+;; Enable EditorConfig mode
+(editorconfig-mode +1)
+
+(defun kotct/editorconfig--disable-smart-tabs-if-indent-tabs-mode-disabled (&optional hash)
```

Comment:
Do you have a suggestion for a better name?  `hash` is a hash passed by EditorConfig when it runs the hooks. It contains the loaded properties from the EditorConfig file.